telemetry/definitions/commonDefinitions.json

```diff
@@ -147,6 +147,17 @@
             "name": "enhancedHealthEnabled",
             "type": "boolean",
             "description": "Whether or not Elastic Beanstalk enhanced health reporting and monitoring is being used"
+        },
+        {
+            "name": "databaseEngine",
```

Comment:
Reason for not using an enum?

Reply:
Since this value comes directly from the service, I don't think there is much value in adding an enum (because we just introduce the potential to have exceptions thrown as new things are added). SecretsManager auth in particular should work with DocumentDB/other engines that we do not support directly right now (with nodes or actions), and so it should be captured properly passing it through. Same with the serviceType, which we should maybe remove the enum list from.